docs/tempo/website/deployment/_index.md

```diff
@@ -6,30 +6,18 @@ weight: 250
 # Deploying Tempo
 
 Tempo can be easily deployed through a number of tools as explained in this document.
-The linked folders contain example deployments of Tempo.  They are a good resource for getting some basic configurations together.
 
-## Docker Compose
+> Note: The Tanka and Helm examples are equivalent.
+> They are both provided for people who prefer different configuration mechanisms.
```

Comment:
`prefer different` -> `prefer a different`

Reply:
`prefer different configuration mechanisms` is correct here